util/opds_writer.py

```diff
@@ -56,13 +56,13 @@ def add_link_to_feed(cls, feed, children=None, **kwargs):
 
     @classmethod
     def add_link_to_entry(cls, entry, children=None, **kwargs):
+        kwargs = dict([unicode(x), unicode(y)] for x, y in kwargs.items())
```

Comment:
I'm not sure about this change. In my circulation branch, an exception was being raised here because the title of a link was a localized string like `l"All Fiction"`. We should be able to handle that, but this is kind of a ham-handed way to manage this problem and I don't understand why my circulation branch was the first time I've ever seen this problem. It seems like `l"All Fiction"` gets passed into here a lot.

Reply:
We haven't been localizing strings in the OPDS feeds yet as far as I know. I don't think this will hurt anything but it might be better to limit it to the title.